app/javascript/dashboard/components/widgets/conversation/bubble/Actions.vue

```diff
@@ -252,6 +247,10 @@ export default {
         return this.sourceId && this.isRead;
       }
 
+      if (this.isAWebWidgetInbox || this.isAPIInbox) {
+        return this.isRead;
```

Comment:
The condition here and the condition this replaces does not match, the computed property checks the message status on the other hand the previous condition compared timestamps. Is this intentional

Reply:
Previously, we find the message read status based on the contact's last seen time. Now, we're updating this flow to mark all messages as read while updating the contact's last seen time. So, every message have the status `read` once the user has viewed it. https://github.com/chatwoot/chatwoot/pull/8092/files#diff-77f099f036eeb74868f3b7f1e330b962f032baabbb4341fbbe7bbce42c6d5eacR5